src/OpenTelemetry.Api.ProviderBuilderExtensions/OpenTelemetry.Api.ProviderBuilderExtensions.csproj

```diff
@@ -8,10 +8,11 @@
     <AnalysisLevel>latest-all</AnalysisLevel>
   </PropertyGroup>
 
-  <!--Do not run ApiCompat as this package has never released a stable version.
+  <!--Do not run ApiCompat nor PackageValidation as this package has never released a stable version.
   Remove this property once we have released a stable version.-->
   <PropertyGroup Condition="'$(TargetFramework)' == 'net6.0'">
     <RunApiCompat>false</RunApiCompat>
+    <SkipPackageValidation>true</SkipPackageValidation>
```

Comment:
By setting this property, you actually don't skip package validation as it still runs. You are skipping package baseline validation by not setting the `PackageValidationBaselineVersion` property.

Reply:
I would recommend to not add any extra properties, if not strictly necessary. I.e. here you could just unset the `PackageValidationBaselineVersion` property: `<PackageValidationBaselineVersion />` and move the src/Directory.Build.targets changes into the Directory.Build.props file.

That said, I'm not 100% sure if that provides a better UX.

Reply:
My attempt to disable the validation by setting `EnablePackageValidation` to false in the same `PropertyGroup` didn't seem to work. This is why I opted to add a new `SkipPackageValidation` property instead.

Reply:
That's because you pass `EnablePackageValidation=true` in as a global property in the YML file. Global properties can't be overwritten. What's the intent here? Do you want to disable package validation completely or just the package baseline validation?

Reply:
The intent is to not perform package validation for some of the packages, specifically for those that do not yet have a stable version to compare with. I'm assuming this is a requirement as this is how it is working for the existing `ApiCompat` workflow. @utpilla could you please comment?

Reply:
If that's the case then we've gone full circle and you need to pass in a custom property in the YML and then set EnablePackageValidation=true based on that custom property. Kind of what you already had. I didn't know about your requirement of being able to disable validation for certain projects.

Reply:
Actually you are right. Since the requirement to have a stable version to compare with only makes sense for package baseline validation, I guess we can keep the other validations in place even in such cases.

Reply:
I just applied your suggestion to use `<PackageValidationBaselineVersion />` instead of a new prop. @ViktorHofer Could you please take another look?

Reply:
Resolving as this was achieved by adding `DisablePackageBaselineValidation` as suggested in another comment.